src/Template/Bake/Middleware/middleware.twig

```diff
@@ -0,0 +1,40 @@
+{#
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         1.3.6
```

Comment:
Well these @since headers are now off.
Also the topic of generated code and licenses has not been resolved for years.

Reply:
You can sync all those once I am finished adding all templates :)

Reply:
Ah, didnt see that comment, yeah we can sync this then afterwards and hopefully remove the license noise in template files.